README.md

```diff
@@ -121,12 +121,14 @@ $ npm install
 
```

Comment:
Would you please add the Selenoid integration details and feature details in Readme.md file.

Also, make sure once you add the selenoid information, you add "* [Selenoid Integratioin](#selenoid-integratioin)" under [Features](#features)

Reply:
Done, added the details